build.gradle.kts

```diff
@@ -22,15 +22,17 @@ dependencies {
 
   implementation("org.flywaydb:flyway-core")
   implementation("io.hypersistence:hypersistence-utils-hibernate-60:3.3.2")
+  implementation("org.hibernate.orm:hibernate-community-dialects:6.2.3.Final")
+  implementation("org.hibernate.orm:hibernate-core:6.2.3.Final")
```

Comment:
hibernate core should be automatically included, not sure you'll need to put it here

Reply:
odd - it initially complained so I had to pull it in.  Removed now as builds fine so was probably a local issue